Detectors/TRD/calibration/macros/plotVdriftExB.C

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file plotVdriftExB.C
+/// \brief Plot calibration values in ccdb for a particluar duration.
+/// \author Felix Schlepper
+
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+// ROOT header
+#include <TFile.h>
+#include <TAxis.h>
+#include <TGraph.h>
+#include <TMultiGraph.h>
+
+// O2 header
+#include "CCDB/BasicCCDBManager.h"
+#include "CCDB/CcdbApi.h"
+#include "DataFormatsTRD/CalVdriftExB.h"
+#include "DataFormatsParameters/GRPECSObject.h"
+
+#include <map>
+#include <string>
+#include <vector>
+#endif
+
+using timePoint = o2::parameters::GRPECSObject::timePoint;
+static std::array<std::vector<std::tuple<double, double, timePoint>>, 540> vmap; // Map holding all values for all detectors.
+static std::array<bool, 540> good;                                               // Marks wether or not a chamber was 'good' for the entire duration.
+
+// Download the values and populate the map.
+void ccdbDownload(unsigned int runNumber, std::string ccdb, timePoint timeOffset)
+{
+  auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
+  ccdbMgr.setURL("http://alice-ccdb.cern.ch/");
```

Comment:
probably you should set the URL to `ccdb` here

Reply:
Didn't you tell me the GRPECSObject are only alice-ccdb not anywhere
else?
Maybe adding a second parameter for the ccdb for GRPECSObjects is better?

Reply:
Yes. Ah, ok, you are changing the URL below to `ccdb` to get the TRD object from the specified path and only the GRP always from alice-ccdb. This I missed. Then this is fine.